jobs/scheduled/check_for_spam_post_voting_comments.rb

```diff
@@ -6,6 +6,7 @@ class CheckForSpamPostsVotingComments < ::Jobs::Scheduled
 
     def execute(args)
       return unless SiteSetting.akismet_enabled?
+      return unless defined?(SiteSetting.post_voting_enabled) && SiteSetting.post_voting_enabled?
```

Comment:
```suggestion
      return unless SiteSetting.respond_to?(:post_voting_enabled) && SiteSetting.post_voting_enabled?
```

Reply:
It won't. I checked with a fresh db prior to writing that line ✨ 

```
[1] pry(main)> defined?(SiteSetting.post_voting_enabled)
=> nil
[2] pry(main)> SiteSetting.post_voting_enabled
NoMethodError: undefined method `post_voting_enabled' for SiteSetting:Class
```

Reply:
O this is interesting, I thought `SiteSetting.post_voting_enabled` will be evaluated first and the error thrown. I guess that is not the case.

Reply:
Oh this is interesting: https://stackoverflow.com/a/44109008